x-pack/legacy/plugins/siem/cypress/integration/lib/timeline/selectors.ts

```diff
@@ -36,3 +36,9 @@ export const TOGGLE_TIMELINE_EXPAND_EVENT = '[data-test-subj="expand-event"]';
 
 /** The body of the timeline flyout */
 export const TIMELINE_FLYOUT_BODY = '[data-test-subj="eui-flyout-body"]';
+
+export const CLOSE_TIMELINE_BTN = '[data-test-subj="close-timeline"]';
+
+export const CLOSE_PROVIDER_BADGE_BTN = '[data-test-subj="closeProviderBadge"]';
```

Comment:
[Per the discussion in #54463](https://github.com/elastic/kibana/pull/54463), this should be:

```
export const CLOSE_PROVIDER_BADGE_BTN = 'button[title="Remove Data Provider"]';
```